src/snowflake/snowpark/_internal/type_utils.py

```diff
@@ -291,7 +291,7 @@ def convert_sp_to_sf_type(datatype: DataType) -> str:
     if isinstance(datatype, StructType):
         if datatype.structured:
             fields = ", ".join(
-                f"{field.name} {convert_sp_to_sf_type(field.datatype)}"
+                f"{field.raw_name} {convert_sp_to_sf_type(field.datatype)}"
```

Comment:
I kinda have the same problem as Afroz's on BCR.. previous we have name upper cased but now we do not, this won't break users?

Reply:
As far as I know nobody is using StructType columns yet due to them requiring structured types be enabled for their account.